src/Jackett.Common/Definitions/yts.yml

```diff
@@ -36,6 +35,10 @@ legacylinks:
   - https://yts.mrunblock.bond/
   - https://yts.nocensor.cloud/
   - https://yts.unblockit.date/
+  - https://yts.unblockninja.com/
+  - https://yts.ninjaproxy1.com/
+  - https://yts.proxyninja.org/
+  - https://yts.torrentbay.st/
```

Comment:
we are happy to use un-official proxies as long as they meet the following criteria:
- that these sites use the same search paths, rows and fields as the official site
- that the download links are known to be real and not malware laden
- that the search results are the same as the official site and that they are at least within a few hours of the official site releases 
- an example of sites that would not be accepted are those that are obvious fakes, such as ones that present a skinned html page that looks similar to the official site but use a different source such as fetching data from TPB API or some other central data repository

In the case of the unblockit and ninja proxies, we are happy to continue using them as un-official proxies as they have been reliably providing the same torrents as the official sites and the indexer works un-altered with their domains.

Reply:
All done now. Kindly review and merge. 😃